### ISSUE TITLE
Variable structure for emissions and energy use of bunker fuels

### ISSUE THREAD

**Issue by danielhuppmann** [IAMconsortium/common-definitions#36]
When merging the ENGAGE and NAVIGATE variable templates, I noticed an issue with the treatment of bunker fuels...

# The problem

Emissions and energy use for international shipping and aviation (aka "bunkers") are not consistently counted in (national) emission inventories and statistics, instead treated as memo items, etc.

# Solution adopted in ENGAGE and NAVIGATE

...and initially transferred into this new variable template:

1. Have international bunkers as a separate sector outside of Transportation, e.g., 
    - Final Energy|Bunkers|International Aviation
    - Final Energy|Bunkers|International Shipping
    - ...

   in contrast to domestic aviation and shipping
    - Final Energy|Transportation|Domestic Aviation
    - Final Energy|Transportation|Domestic Shipping

    and sub-categories, parallel for emissions...

2. There are several "memo items" for comparability with relevant statistics
    - Emissions|CO2 (w/o bunkers)
    - Emissions|CO2|Energy|Demand|Transportation (w/ bunkers)
    - Final Energy (w/o bunkers)
    - Final Energy|Transportation (w/ bunkers)

# Problems going forward

1. As a consequence, energy use from green fuels (e.g., battery, hydrogen planes) is accounted within the bunkers category, even though they are not "bunker fuels"...
2. The "memo items" are once "with bunkers", once "without bunkers", which may be confusing...

# Alternative proposal

1. Move international transportation energy use and emissions back into the "Transportation" variable tree
2. Make memo items consistently as "w/o bunkers"

To be explicit, the proposal is:

- Final Energy
- Final Energy (w/o bunkers) [Memo Item]
- Final Energy|Transportation
- Final Energy|Transportation (w/o bunkers) [Memo Item]
- Final Energy|Transportation|Domestic Aviation
- Final Energy|Transportation|Domestic Shipping
- Final Energy|Transportation|International Aviation
- Final Energy|Transportation|International Shipping
- Final Energy|...

and sub-categories, parallel for emissions...

And it should be clarified whether the memo items includes non-fossil energy use of international shipping and aviation...

Any comments @IAMconsortium/common-definitions-coordination?

**Comment by strefler** [comment]
The reasons for the w/ or w/o bunkers is historic: We wanted to avoid keeping the same variable name, but changing the definition either from w/ to w/o bunkers or the other way around. So in order to keep backwards compatibility with older reporting templates, I suggest to keep this.
I see the problem with green fuels being accounted as bunker fuels. However, one of the reasons why we decided to split bunkers and transport is that it makes regional aggregation easier. Bunkers are sometimes only reported on the global level, and if they are included in Transport, then the sum over regions would not match the global total. To me the current split seems more intuitive, and I would rather live with green fuels being accounted as bunker fuels

**Comment by danielhuppmann** [comment]
I'm not a supporter of this structure (because I think that it can cause some confusion by scenario users if "transportation" does not include international aviation and shipping)...

But for consistency, I suggest to at least change the descriptions such that this extra sector "Bunkers" refers to all energy consumption in international aviation and shipping and make that also clear in the description of "Transportation". See #36 for an explicit proposal.

**Comment by robertpietzcker** [comment]
I agree, Daniel, to me also the more logical structure would be to have intAv&intShip included in Transportation and not as a separate subsector, as they have the same purpose.
 
But I also see the challenge that most national statistics have them separate from their reporting, so for fast comparison to recent data from some national statistics bureau (that is not yet included in the analysis scripts, where it would be easy to recalculate the "with bunker fuels" values) it is helpful to follow that approach. 

so I am undecided which one I would find better. 

(actually, once intShipping is fully included in the EU-ETS in 2024, I am wondering if the reporting standard in the EU will not also change. and in 2026 there is the CORSIA review, and I wouldn't be too surprised if CORSIA doesn't live up to expectations and intAv is then also included fully in the EU-ETS...)
